lib/Jit/LLILCJit.cpp

```diff
@@ -303,6 +283,16 @@ CorJitResult LLILCJit::compileMethod(ICorJitInfo *JitInfo,
 
       // Tell the CLR that we've successfully generated code for this method.
       Result = CORJIT_OK;
+
+      if (Context.Options->DumpLevel >= ::DumpLevel::SUMMARY) {
+         errs() << "LLILC compiled: " 
+                << ", Entry = " << *NativeEntry 
+                << ", End = " << (*NativeEntry + *NativeSizeOfCode)
+                << ", size = " << *NativeSizeOfCode 
+                << " method = " << Context.MethodName 
+                << '\n';
+      }
+
```

Comment:
Why not break this into two lines?

```
errs() << "Successfully read " << Context.MethodName << '\n';
errs() << "\tEntry: " << *NativeEntry << ", End: " << (*NativeEntry + *NativeSizeOfCode) << ", Size: " << *NativeSizeOfCode << '\n';
```

Keeping the original format in the first line would eliminate the need to update any of the scripts.